telegram/ext/_application.py

```diff
@@ -113,12 +113,31 @@ class Application(Generic[BT, CCT, UD, CD, BD, JQ]):
         * Initialization is now done through the :class:`telegram.ext.ApplicationBuilder`.
         * Removed the attribute ``groups``.
 
+    Args:
```

Comment:
mh, why did you document the arguments? this class should not be instantiated manually but via the `ApplicationBuilder`

Reply:
oh, felt weird not to have an Args section. Maybe link to the ApplicationBuilder in the Args section? That way it becomes a bit more clear to the users

Reply:
you mean like [this](https://github.com/python-telegram-bot/python-telegram-bot/blob/1bbe39df051bafd1b181262554b2ec6b3dafe733/telegram/ext/_application.py#L107-L114)? ;P

Reply:
yeah like move that Note to Args section

Reply:
ah. mh, I don't see a big difference between it being a note and it bein in the `args` section … but I'm also not agains it. you can try and see what's possible here - not sure how well that plays with sphinx's layout …